http.md

```diff
@@ -0,0 +1,340 @@
+# JSON RPC HTTP Methods
```

Comment:
can you add this in a folder called `json_rpc` to separate from protos

Reply:
done